src/LocalizationWrapper.jsx

```diff
@@ -1,33 +1,36 @@
-import React, {Component, PropTypes} from 'react';
+import React, { Component, PropTypes } from 'react';
 import get from 'lodash.get';
 import defaultLocalizer from './util/localize-formatter';
 
 export default (ComposedComponent, messages = {}, customLocalizer) => {
   class ConfigureLocalization extends Component {
-    constructor(props) {
-      super(props);
+    static localize(key, values) {
```

Comment:
This wrapper is just a way to avoid using `<Localization/>` around your app, so you could do something like `export default Localized(App)` which would setup the context localize stuff for the app children.

This _function_ here is used when `this.context.localize` is used, as this function is what is passed down through context. I could simply have moved it to `getChildContext()` too tho.